### DISCUSSION TITLE
ValueError: Normal distribution got invalid loc parameter

### DISCUSSION THREAD

**Discussion by arezoosamimi** [google/lightweight_mmm#94] [General]
While running the model.fit, sometimes I got this error: "ValueError: Normal distribution got invalid loc parameter". I've seen that using different period of time might solve the problem but not sure why I'm getting this error. 
Could you please help me to understand the underlying issue? and how I can fix the problem? Thank you

**Comment by queili** [comment]
Hello @ArziAlo!
I had the same issue when some of my channels had data points in train set only or test set only.
For example, if channel A started to run only at the end of the period, when you break into training and testing it will only be on one of them.
You can try to visualize the time series of each channel and remove those that only appear at the beginning or end of the data period.

**Comment by michevan** [comment]
thanks for flagging this and thanks for the helpful answer @kayleighdp6! We'll try to reproduce this and see if we can add some guardrails or something to help with it.

**Reply by danivapp** [reply]
Hello @michevan, 

i came across an erorr that states "ValueError: Normal distribution got invalid scale parameter.", is this related?

**Comment by michevan** [comment]
could it be related to non-positive numbers in your media priors? c.f. [this discussion](https://github.com/google/lightweight_mmm/issues/115).
